test/integration/connect/envoy/run-tests.windows.sh

```diff
@@ -332,14 +332,7 @@ function verify {
 
   # need to tell the PID 1 inside of the container that it won't be actual PID
   # 1 because we're using --pid=host so we use TINI_SUBREAPER
-  if docker.exe run --name envoy_verify-${CLUSTER}_1 -t \
-    -e TINI_SUBREAPER=1 \
-    -e ENVOY_VERSION \
-    $WORKDIR_SNIPPET \
-    --pid=host \
-    $(network_snippet $CLUSTER) \
-    bats-verify \
-    --pretty ${CLUSTER}/bats ; then
+  if docker.exe exec -i ${SINGLE_CONTAINER_BASE_NAME}-${CLUSTER}_1 bash -c "/c/bats/bin/bats --pretty /c/workdir/${CLUSTER}/bats" ; then
```

Comment:
Are not needed the environments variables TINI_SUBREAPER and ENVOY_VERSION?
I noted that we have bats in the Path, so we can use it directly without indicate its path

nit: I see a little more legible the line break to split the commands/parameters and to be consistent with others docker exec/run commands

```suggestion
  if docker.exe exec -t $CONTAINER_NAME bash.exe -c "TINI_SUBREAPER=1 \
                                                      ENVOY_VERSION=${ENVOY_VERSION} \
                                                      bats --pretty /c/workdir/${CLUSTER}/bats"; then
```